packages/cli/src/tokens/build/configs.ts

```diff
@@ -2,14 +2,21 @@ import { expandTypesMap, register } from '@tokens-studio/sd-transforms';
 import type { ThemeObject } from '@tokens-studio/types';
 import * as R from 'ramda';
 import StyleDictionary from 'style-dictionary';
-import type { Config, LogConfig, TransformedToken } from 'style-dictionary/types';
+import type SD from 'style-dictionary/types';
```

Comment:
Suggest keeping deconstructed types to stay consistent with use elsewhere, both in CLI and other packages.

Reply:
I just REALLY don't like that Style Dictionary's type is simply called `Config`, I found it very confusing to see a bare `Config` in the type signatures

Reply:
How do you feel about this?
```ts
import type { Config as StyleDictionaryConfig, TransformedToken } from 'style-dictionary/types';
// ...
type GetStyleDictionaryConfig = (
  options: ThemePermutation & {
    outPath?: string;
  },
) => StyleDictionaryConfig;

```

Reply:
Looks good :)